frontend/src/app/assess/[projectId]/fairness-bias/report/page.tsx

```
@@ -326,14 +325,14 @@ export default function FairnessBiasReport() {
                           {/* Reasoning */}
                           {evaluation.reasoning && (
                             <details className="group">
-                              <summary className="cursor-pointer text-sm font-medium text-purple-600 dark:text-purple-400 hover:text-purple-700 dark:hover:text-purple-300 flex items-center gap-2">
+                              <summary className="cursor-pointer text-sm font-medium text-primary hover:text-primary/80 flex items-center gap-2">
                                 <span>View AI Analysis</span>
                                 <svg className="w-4 h-4 transition-transform group-open:rotate-180" fill="none" stroke="currentColor" viewBox="0 0 24 24">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)